src/EditorFeatures/Core/Implementation/KeywordHighlighting/HighlightingService.cs

```diff
@@ -27,6 +27,11 @@ internal class HighlightingService : IHighlightingService
         public IEnumerable<TextSpan> GetHighlights(
              SyntaxNode root, int position, CancellationToken cancellationToken)
         {
+            if (root == null)
```

Comment:
Couldn't figure out why this was `null` from the dump though.

Reply:
Tagging @heejaechang - related to https://github.com/dotnet/roslyn/pull/32388? Are both of them same underlying issues (root == null?)

Reply:
definitely seems unfortunate that we don't know why.  that said, the fix seems reasonable.

Reply:
Ah, seems @heejaechang was trying to address the same bug :)